### DISCUSSION TITLE
Purple Graphs [3.5 inch]

### DISCUSSION THREAD

**Discussion by v1sionaire** [mathoudebine/turing-smart-screen-python#628] [Themes]
Layout is based on [Classic Black Theme](https://github.com/mathoudebine/turing-smart-screen-python/discussions/609).

Added:
- Purple colors
- Resized items
- Changed fonts
- Added icons
- Added graph meshes

![image](https://github.com/user-attachments/assets/bd7864cc-c5a8-493f-a5f1-c84dbaa9996b)
![image](https://github.com/user-attachments/assets/65f717d3-da74-4621-80f7-39c19307ea2e)

Download:
[PurpleGraphs.zip](https://github.com/user-attachments/files/18240490/PurpleGraphs.zip)

**Comment by sysyphus** [comment]
Nice!!

**Comment by matthewr88** [comment]
I've been using this theme for months. But now I've updated to 3.9.7 with the new version of LHM using pawn.io I have to force it to use LHM otherwise the GPU stats don't appear. But also, after some time the CPU graph just freezes and doesn't update. I have to reset turing to get it back running. I'm running an intel 7600k with iGPU. I'm guessing this is going to be an LHM issue... but any ideas?
